django/template/loader_tags.py

```diff
@@ -170,10 +170,14 @@ def render(self, context):
         if not callable(getattr(template, 'render', None)):
             # If not, try the cache and get_template().
```

Comment:
Comment is (I think) now out of date, `get_template()` -> `select_template()`

Reply:
well spotted, it is

Reply:
@kezabelle Thanks.